app/models/concerns/profile_tailoring.rb

```diff
@@ -23,4 +33,27 @@ def added_rules
   def removed_rules
     parent_profile.rules - rules
   end
+
+  def update_os_minor_version(version)
+    return unless version && os_minor_version.empty?
+
+    update!(os_minor_version: version)
+  end
+
+  private
+
+  def in_account(account, policy)
+    Profile.find_by(account: account, ref_id: ref_id,
+                    policy: policy,
+                    benchmark_id: benchmark_id)
+  end
```

Comment:
This should move to profile_searching.rb probably. It's not just for tailoring. The canonical_for_os scope should also be in profile_searching.rb, something we overlooked when it was added, but it should give you more space in profile.rb to keep rubocop happy with this change.